src/main/kotlin/org/jitsi/jicofo/conference/colibri/v2/Colibri2Session.kt

```diff
@@ -48,18 +49,18 @@ internal class Colibri2Session(
     val colibriSessionManager: ColibriV2SessionManager,
     val bridge: Bridge,
     parentLogger: Logger
-) {
+) : CascadeNode<Colibri2Session, Colibri2Session.Relay> {
     private val logger = createChildLogger(parentLogger).apply {
         bridge.jid.resourceOrNull?.toString()?.let { addContext("bridge", it) }
     }
     private val xmppConnection = colibriSessionManager.xmppConnection
     val id = UUID.randomUUID().toString()
     /**
      * Save the relay ID locally since it is possible for the relay ID of the Bridge to change and we don't want it to
-     * change in the context of a session. We maintain the invariant that whenever a a conference has multiple sessions,
+     * change in the context of a session. We maintain the invariant that whenever a conference has multiple sessions,
      * they all have non-null relay IDs.
      */
-    val relayId: String? = bridge.relayId
+    override val relayId: String? = bridge.relayId
```

Comment:
We could use `bridge.jid` as the relay instead (which is already non-nullable)

Reply:
This would be a change from what we do today, yes? And thus potentially more disruptive of code I'm already feeling like I have to be fairly delicate about...